docs/index.rst

```diff
@@ -11,5 +11,5 @@ Also see the `Cython project homepage <https://cython.org/>`_.
    src/userguide/index
    src/userguide/glossary
    src/reference/index
-   Contributing <CONTRIBUTING>
+   src/devguide/index
```

Comment:
This is actually a special file in Github that is supposed to be linked from the issues and pull request forms as guidelines. Although I fail to see the link, now that I checked. Maybe the `.rst` extension is getting in the way?

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors